modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.server/src/main/scala/edu/gemini/seqexec/web/server/http4s/SeqexecUIApiRoutes.scala

```diff
@@ -103,7 +103,8 @@ class SeqexecUIApiRoutes(auth: AuthenticationService, q: engine.EventQueue) exte
           } yield (obsId, s)
 
           r match {
-            case \/-((i, s)) => Ok(List(Sequence(i.stringValue(), SequenceState.NotRunning, "F2", s.toSequenceSteps, None)))
+            case \/-((i, s)) => SeqexecEngine.load(i) *>
```

Comment:
I'm not sure what the `load` call does. I'd think you want the output to be used in the response, though it is a `Unit`